README.md

```diff
@@ -476,7 +475,7 @@ Features
       TeamSpeak2 server statistics.
 
     - ted
-      Plugin to read values from `The Energy Detective' (TED).
+      plugin to read values from "the energy detective" (ted).
```

Comment:
Other plugin entries are capitalized, and according to quick Googling, TED **is** also capitalized:
```suggestion
      Plugin to read values from "The Energy Detective" (TED).
```